shellbase-core/src/main/scala/com/sumologic/shellbase/ShellBase.scala

```diff
@@ -318,12 +318,17 @@ abstract class ShellBase(val name: String) {
 
   private def runSingleTokenizedCommand(tokens: List[String]): Boolean = {
     val out = rootSet.executeLine(tokens)
+    val cmd = tokens.mkString("'", " ", "'")
     val msg = if (out) {
-      s"[$name] Command finished successfully"
+      s"Command finished successfully: $cmd"
     } else {
-      s"[$name] Command failed"
+      s"Command failed: $cmd"
+    }
+    // NOTE(konstantin, 2017-55-03): this is a hackish way to suppress notifications from the notifications command
+    // itself. We should probably introduce a notion of "quiet" commands instead?
+    if (!tokens.headOption.contains(notificationCommandSet.name)) {
```

Comment:
What about the aliases? wondering whether we should make this less hackish and add a function to `ShellCommand` to do this comparison.